src/main/java/de/blablubbabc/insigns/InSigns.java

```diff
@@ -95,8 +95,8 @@ void updateNearbySignsDelayed(Player player, int delayTicks) {
 		}, delayTicks);
 	}
 
-	SignSendEvent callSignSendEvent(Player player, Location location, String[] rawLines) {
-		SignSendEvent signSendEvent = new SignSendEvent(player, location, rawLines);
+	SignSendEvent callSignSendEvent(Player player, Location location, String[] rawLineFront, String[] rawLineBack) {
```

Comment:
Can you name this "rawLine**s**Front/Back"?

Reply:
fixed